src/pages/Strategies/Strategies.js

```diff
@@ -199,7 +199,11 @@ const StrategiesPage = ({
     setSectionErrors({})
     setNextStrategyToOpen(null)
     setStrategyDirty(false)
-    setIDEcontent(strategyToLoad.strategyContent)
+    if (!_isEmpty(strategyToLoad?.strategyContent)) {
+      setIDEcontent(strategyToLoad.strategyContent)
+    } else {
+      setIDEcontent({})
+    }
```

Comment:
```suggestion
    if (_isEmpty(strategyToLoad?.strategyContent)) {
      setIDEcontent({})
    } else {
      setIDEcontent(strategyToLoad.strategyContent)
    }
```

I think it will be easier to read the code if we use `_isEmpty` instead of `!_isEmpty` and just swap the contents of `if/else` blocks.

Reply:
you are right. thanks!